### PR TITLE
feat: Adds function docstring for get_pollution_forecast

### LINKED ISSUE #7: Docstring for get_pollution_forecast()
Fetches air pollution data for the next 5 days for a specific location. The function returns a time series plot of the predicted pollution levels.

### PR DESCRIPTION
**Pull request by christopheralex**
- [x] Write docstring for get_pollution_forecast()

Observations :
- The API returns multiple pollutants. We would have to decide on one of them
     - If we are going to use a single pollutant,
     - Multiple graphs for each pollutant,
     - Use a facet
- All pollutant concentration's have a scale associated with it. ( Include as color ) 
    - number of classes to be used could be modified for better visualization.

### CODE REVIEW

File: src/AirPyllution/airpyllution.py [hunk 1]
```diff
@@ -67,4 +67,31 @@ def get_air_pollution(lat, lon, api_key):
     Examples
     --------
     >>> get_air_pollution(49.2497, -123.1193, "APIKEY_example")
+    """
+
+def get_pollution_forecast(lat, lon, api_key):
+    """Returns a time series plot showing predicted pollutant levels for the next 5 days
```
Comment (danfke) [inline src/AirPyllution/airpyllution.py]: Forgot a period at the end lol

File: src/AirPyllution/airpyllution.py [hunk 2]
```diff
@@ -67,4 +67,31 @@ def get_air_pollution(lat, lon, api_key):
...
+        geographical latitude coordinate for the location
+    lon : float
+        geographical longitude coordinate for the location
+    api_key: string
+        OpenWeather API key
+
+    Returns
+    -------
+    altair.Chart
+        altair chart object with the x axis as time/UNIX timestamp and
```
Comment (danfke) [inline src/AirPyllution/airpyllution.py:91]: I probably should add more description to my docstring to match yours

File: src/AirPyllution/airpyllution.py (hunk 1 above)
Comment (christopheralex) [reply src/AirPyllution/airpyllution.py]: Do you want to me to add ` for the next 5 days from the current date ` ? Or is it a period period `.` ?

File: src/AirPyllution/airpyllution.py (hunk 2 above)
Comment (christopheralex) [reply src/AirPyllution/airpyllution.py:91]: I don't think its an absolute necessity as these docstrings are all in some way abstract and might change later.

File: src/AirPyllution/airpyllution.py (hunk 1 above)
Comment (danfke) [reply src/AirPyllution/airpyllution.py]: Yah I meant a literal period `.`

Comment (christopheralex) [reply src/AirPyllution/airpyllution.py]: Done.

**Review by mel-liow — APPROVED** [review]
Looks good!

**Pull request merged**